beets/util/__init__.py

```diff
@@ -737,10 +737,26 @@ def cpu_count():
         return 1
 
 
+def convert_command_args(args):
+    """Convert command arguments to bytes on Python 2 and strings on
+    Python 3."""
+    def convert(arg):
+        if six.PY2:
+            if isinstance(arg, six.text_type):
+                arg = arg.encode(arg_encoding())
+        else:
+            if isinstance(arg, bytes):
+                arg = arg.decode(arg_encoding(), 'surrogateescape')
+        return arg
+
+    return [convert(a) for a in args]
+
+
```

Comment:
if you're still here, can you cancel https://ci.appveyor.com/project/beetbox/beets/build/842  ?  they broke the default behaviour of `cinst` so the build is hung.